test/helpers/Factory.ts

```diff
@@ -290,7 +290,8 @@ const createNetwork = async (
     networkSettings: NetworkSettings,
     masterVault: MasterVault,
     externalProtectionVault: ExternalProtectionVault,
-    bntPoolToken: PoolToken
+    bntPoolToken: PoolToken,
+    arbContractAddress: string
```

Comment:
should follow the same format as the others, not just a string address but use the specific BancorArbitrage type etc.

Reply:
Discussed to leave it due to complications requiring to change all BancorNetwork deployment tests.